server/src/com/cloud/servlet/ConsoleProxyServlet.java

```diff
@@ -16,35 +16,6 @@
 // under the License.
 package com.cloud.servlet;
 
-import java.io.IOException;
```

Comment:
There seem to be a lot of unnecessary changes in this file. Can you please check this and make sure you only include the changes that are needed